code/datums/components/horror_aura.dm

```diff
@@ -0,0 +1,10 @@
+/datum/component/horror_aura/Initialize()
+	if(ismob(parent))
+		RegisterSignal(parent, .proc/aura_effect)
```

Comment:
you don't actually need a signal
a signal means you want to listen to something 
for you, you realistically just need to 
START_PROCESSING(SSobj, src)